human_experiments/lab_software/tomcat-baseline-tasks/tasks/rest_state/server_rest_state.py

```diff
@@ -19,13 +32,30 @@ def run(self):
 
         print("[STATUS] Running rest state")
         send(self._to_client_connections, data)
+
+        log_first_timestap = True  # Log timestamp as soon as the experiment starts
+
+        if log_first_timestap == True:
```

Comment:
we always log_first_timestamp, so we do not need the variable `log_first_timestamp` nor if statement. Remove the if statement and always run the code in it.